CMakeLists.txt

```diff
@@ -3,12 +3,23 @@ cmake_minimum_required(VERSION 3.16.8)
 project(hipify-clang)
 
 include(GNUInstallDirs)
-find_package(LLVM REQUIRED CONFIG PATHS ${CMAKE_PREFIX_PATH})
 
-message(STATUS "Found LLVM ${LLVM_PACKAGE_VERSION}:")
-message(STATUS "   - CMake module path: ${LLVM_CMAKE_DIR}")
-message(STATUS "   - Include path     : ${LLVM_INCLUDE_DIRS}")
-message(STATUS "   - Binary path      : ${LLVM_TOOLS_BINARY_DIR}")
+option(HIPIFY_INCLUDE_IN_HIP_SDK "Include HIPIFY in HIP SDK" OFF)
+
+if(HIPIFY_INCLUDE_IN_HIP_SDK)
+  if(NOT WIN32)
+    message(FATAL_ERROR "HIPIFY_INCLUDE_IN_HIP_SDK is only supported on Windows")
+  endif()
+endif()
+
+if(NOT HIPIFY_INCLUDE_IN_HIP_SDK)
```

Comment:
Now the below `include(AddLLVM)` doesn't work if `HIPIFY_INCLUDE_IN_HIP_SDK` is on.

Reply:
This is not true. I am doing monolithic build, so that works fine.

Reply:
I meant the case when `HIPIFY_INCLUDE_IN_HIP_SDK` is ON but not in the monolithic build. I'll exclude this case in a separate change.